spec/origin_info_spec.rb

```diff
@@ -0,0 +1,146 @@
+describe "computations from /originInfo field" do
+
+  let(:smods_rec) { Stanford::Mods::Record.new }
+
+  let(:mods_origin_info_start_str) { "<mods xmlns=\"#{Mods::MODS_NS}\"><originInfo>" }
```

Comment:
I can't tell, off-hand, the provenance of these.. I assume this file has combined old tests that used `mods_origin_info_start_str`, and new tests that use  `mods_origin_info`?